tests/foreman/cli/test_activationkey.py

```diff
@@ -183,7 +183,7 @@ def test_positive_create_with_usage_limit_default_explicitly(self):
 
         @Assert: Activation key is created
         """
-        new_ak = self._make_activation_key({u'unlimited-content-hosts': '1'})
+        new_ak = self._make_activation_key()
```

Comment:
What do you mean by 'not supported'?
Here's what the doc says:
```
hammer -u admin -p changeme activation-key create --help
...
Options:
...
 --unlimited-content-hosts                           Set content hosts max to unlimited
```
No `true/false/yes/no` values anymore, so that's a bool flag. Maybe the doc is outdated? Nope:
```
hammer -u admin -p changeme activation-key create --organization-id 1 --name testak1 --unlimited-content-hosts
Activation key created
```
Maybe hammer just ignores invalid field? The answer is No:
```
hammer -u admin -p changeme activation-key create --organization-id 1 --name testak1 --unlimited-content-hosts1
Could not create the activation key:
  Error: Unrecognised option '--unlimited-content-hosts1'
  
  See: 'hammer activation-key create --help'
```
Maybe the field is accepted but actually not doing anything? No, validation is on the place, for example you can't define limit and unlimited at the same time:
```
hammer -u admin -p changeme activation-key create --organization-id 1 --name testak1 --unlimited-content-hosts --max-content-hosts 1
Could not create the activation key:
  Error: You can't set all options --unlimited-content-hosts, --max-content-hosts at one time
  
  See: 'hammer activation-key create --help'
```

So why do you think it's not supported anymore? It's working ok, so i can't see a reason to cut the test.
And also - please take a look at the previous test, `test_positive_create_with_usage_limit_default`. Your change will make these 2 tests identical.